### ISSUE TITLE
The MobileNet-v2 pre-trained model seems to have some problems

### ISSUE THREAD

**Issue by CGod1991** [Tencent/PocketFlow#154]
Hi:
I have followed the usage steps of UniformQuantTFLearner, and used the MobileNet-v2 model which download from the pre-trained model page, but i got this error:

Restoring from checkpoint failed. This is most likely due to a Variable name or other graph key that is missing from the checkpoint. Please ensure that you have not altered the graph expected based on the checkpoint. Original error:

Key model/MobilenetV1/Conv2d_0/BatchNorm/beta not found in checkpoint
         [[Node: model/save/RestoreV2 = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_model/save/Const_0_0, model/save/RestoreV2/tensor_names, model/save/RestoreV2/shape_and_slices)]]

It seems like that the pre-trained MobileNet-v2 model has some problems.

PS: This is my command: 
sh scripts/run_local.sh nets/mobilenet_at_ilsvrc12_run.py --mobilenet_version=2 --nb_epochs_rat=0.2 -n=1 --learner=uniform-tf

**Comment by jiaxiang-wu** [comment]
This is due to a bug in our execution scripts (`run_local.sh` / `run_docker.sh` / `run_seven.sh`). The execution script will search for the "-n=x" option and ignore all the options before it. Therefore, `--mobilenet_version=2` is ignored so you can see that the program is trying to restore a MobileNet-v1 models (instead of MobileNet-v2) from MobileNet-v2 checkpoint files, which leads to an error.

TODO: fix the argument parsing issue in execution scripts.

**Comment by jiaxiang-wu** [comment]
@CGod1991 Could you please take a look at PR #157 and verify whether your issue has been resolved?

**Comment by florinmatei** [comment]
I'm having the same problem. I checked the #157 as you suggested and I have the latest changes of  run_local.sh that deals with the extra arguments but same issue. My issue is with the mobilenet-v1 which gives me the error below:

  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1770, in __init__
    self._traceback = tf_stack.extract_stack()

NotFoundError (see above for traceback): Restoring from checkpoint failed. This is most likely due to a Variable name or other graph key that is missing from the checkpoint. Please ensure that you have not altered the graph expected based on the checkpoint. Original error:

Key model/MobilenetV1/Conv2d_0/BatchNorm/beta not found in checkpoint
	 [[node model/save/RestoreV2 (defined at /home/ubuntu/ai-platform2-poc/PocketFlow/learners/discr_channel_pruning/learner.py:204)  = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_model/save/Const_0_0, model/save/RestoreV2/tensor_names, model/save/RestoreV2/shape_and_slices)]]

The command is similar to the one in the original post but instead I'm using --mobilenet_version=1

Is there a fix for this?
Thank you!

**Comment by florinmatei** [comment]
@CGod1991 were you able to fix your problem?
